specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2021-01-01-preview/MigrateFromSmartDetections.json

```diff
@@ -0,0 +1,304 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-01-01-preview",
+    "title": "Azure Alerts Management Service Resource Provider",
+    "description": "APIs for managing a migration from Smart Detection to smart alerts."
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection": {
+      "post": {
+        "tags": [
+          "MigrateFromSmartDetection"
+        ],
+        "description": "Starts an asynchronous migration process of Smart Detection to smart alerts in an Application Insights resource.",
+        "operationId": "MigrateFromSmartDetection_StartMigration",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "smartDetectionMigrationRequest",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SmartDetectionMigrationRequest"
+            },
+            "description": "The parameters controlling the migration process."
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Migration request was accepted. Processing will start promptly.",
+            "schema": {
+              "$ref": "#/definitions/MigrationStatusResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/MigrationErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Migration Triggering": {
+            "$ref": "./examples/migrateFromSmartDetectionStartMigration.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection/{migrationId}": {
```

Comment:
>"/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrateFromSmartDetection/{migrationId}" [](start = 4, length = 110)

Just had a second thought here, do you still need this endpoint? your POST action seems similar to this one, POST action should expose an action name: e.g. restart, there's nothing else after the action name.

You can model this like this:

"/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrations" -> to return a list

"/subscriptions/{subscriptionId}/providers/Microsoft.AlertsManagement/migrations/{migrationId}" -> to return a single item